source/config.go

```diff
@@ -94,6 +101,9 @@ func ParseConfig(cfgRaw map[string]string) (Config, error) {
 	if cfg.Table == "" {
 		return Config{}, requiredConfigErr(ConfigKeyTable)
 	}
+	if cfg.OrderingColumn == "" {
```

Comment:
I've decided to add this field to the order not only for the `trigger` CDC mode, but also for `logrepl`.
I think it's quite important because if sorting is not chosen, the rows will be returned in an unspecified order.
https://www.postgresql.org/docs/current/queries-order.html